release/salt-yum-defs/wrangler.auto.tfvars

```diff
@@ -1,7 +1,9 @@
 salt_version = "2016.11.9"
 
 extra_salt_versions = [
-  "2017.7.5",
+  "2016.11.9",
```

Comment:
Why do you have `2016.11.9` in `extra_salt_versions` as well?

Reply:
Just clarity, and aligning with the dev pipeline. The formula de-dupes the versions.